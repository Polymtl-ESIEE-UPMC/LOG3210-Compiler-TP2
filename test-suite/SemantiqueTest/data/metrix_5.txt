Declaration
num a;
num b;
num c;
EndDeclaration

a = 1;
b = 10;
c = 4;
while(true) {
    while(false) {
        while(a == b) {
            if (a < c) {
                a = b + c * 4;
            }
        }
    }
}
