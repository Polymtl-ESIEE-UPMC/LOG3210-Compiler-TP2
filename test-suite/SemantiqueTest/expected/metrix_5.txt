{VAR:3, WHILE:3, IF:1, OP:4}